src/controllers/users.controller.ts

```diff
@@ -119,203 +118,131 @@ export const authenticate = async (req: Express.Request, res: Express.Response,
     }
 };
 
-export const getUsers: RequestHandler = async (req, res, errFunc) => {
+export const getUsers: RequestHandler = async (req, res) => {
     try {
-        const users = await usersDB.findAll();
```

Comment:
need to strip password hash from this data we return. can be done in the db query.